leaked_credential_check.go

```diff
@@ -0,0 +1,190 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/goccy/go-json"
+)
+
+type LeakedCredentialCheckGetStatusParams struct{}
+
+type LeakedCredentialCheckStatus struct {
+	Enabled bool `json:"enabled"`
```

Comment:
https://github.com/cloudflare/cloudflare-go/blob/master/docs/conventions.md#booleans

```suggestion
	Enabled *bool `json:"enabled"`
```